ImageClassification/ViewControllers/ViewController.swift

```diff
@@ -235,9 +235,11 @@ extension ViewController: CameraFeedManagerDelegate {
 
 	fileprivate func deleteCharacter() {
 		DispatchQueue.main.async {
-			var text = self.outputTextView.text
-			text?.removeLast()
-			self.outputTextView.text = text
+            if(self.outputTextView.text != ""){
```

Comment:
Swift lint warning. I think you're on the right track trying to guard the delete but I don't think you're doing it the right way according to lint...
```swift
Control Statement Violation: `if`, `for`, `guard`, `switch`, `while`, and `catch` statements shouldn't unnecessarily wrap their conditionals or arguments in parentheses. (control_statement)
```

Reply:
turns out it just didn't like the parenthesis... [commit](https://github.com/Capstone-Projects-2020-Spring/iASL-iOS/pull/17/commits/ac302deae03bd3801f6f423d128bee2056f3f3d1)